src/ViewOutput.js

```diff
@@ -0,0 +1,28 @@
+import { Console } from '@woowacourse/mission-utils'
+import { showProgressing, gameMessage } from './constants.js';
+
+export default class ViewOutput {
+    
+    static printProgressResult(carList) {
+        carList.forEach(car => {
+            this.printProgressDash(car.getName(), car.getProgress());
+        });
+    }
+
+    static printProgressDash(carName, carProgress) {
+        let progressBar = `${carName} : `;
+        for(let i = 1; i<=carProgress;i++) {
+            progressBar += `${showProgressing.progressBar} `;
+        }
+        Console.print(progressBar);
+    }
```

Comment:
```suggestion
    static printProgressDash(carName, carProgress) {
        const progressBar = `${carName} : ${showProgressing.progressBar.repeat(carProgress)}`;
        Console.print(progressBar);
    }
```

위와 같이 작성한다면, 간단하면서도 `let` 사용을 피할 수 있을 것 같아요 :)

Reply:
[Airbnb style guide](https://github.com/airbnb/javascript?tab=readme-ov-file#references--prefer-const)

Reply:
와 저기에 repeat를 쓸 생각은 못했네요!
알려주셔서 감사해요!